### PR TITLE
Add docs about KV metadata

### PR DESCRIPTION
**Pull request by ispivey**
Taking over #638

### CODE REVIEW

File: workers-docs/src/content/reference/apis/kv.md [hunk 1]
```diff
@@ -101,6 +101,15 @@ You can also [write with an expiration on the command line via
 Wrangler](/tooling/wrangler/kv_commands/#kv-key) or [via the
 API](https://api.cloudflare.com/#workers-kv-namespace-write-key-value-pair).
 
+## Metadata
+
+You may want to associate some metadata with a key-value pair. To do this, you
```
Comment (victoriabernard92) [inline workers-docs/src/content/reference/apis/kv.md:106]:
```suggestion
To associate some metadata with a key-value pair
```

File: workers-docs/src/content/reference/apis/kv.md [hunk 2]
```diff
@@ -101,6 +101,15 @@ You can also [write with an expiration on the command line via
...
+can set `metadata` to any arbitrary object (must serialize to JSON) in the put
```
Comment (victoriabernard92) [inline workers-docs/src/content/reference/apis/kv.md:107]:
```suggestion 
set `metadata` to any arbitrary object (must serialize to JSON) in the put
```

File: workers-docs/src/content/reference/apis/kv.md [hunk 3]
```diff
@@ -101,6 +101,15 @@ You can also [write with an expiration on the command line via
...
+options object on a `put` call. You would run the following to add associated
```
Comment (victoriabernard92) [inline workers-docs/src/content/reference/apis/kv.md:108]:
```suggestion
options object on a `put` call. Usage in Worker script:
```

File: workers-docs/src/content/reference/apis/kv.md [hunk 4]
```diff
@@ -101,6 +101,15 @@ You can also [write with an expiration on the command line via
...
+metadata on a call to `put`:
```
Comment (victoriabernard92) [inline workers-docs/src/content/reference/apis/kv.md:109]:
```suggestion
```

**Comment by ispivey** [comment]
Oops, just realized I poked Victoria for review while she's on holiday. I'm going to assume that addressing her comments is sufficient to merge 👍

**Pull request merged**